### PR TITLE
Turn bin/reset_dummy into lib/tasks/dummy_reset.rake

### PR DESCRIPTION
**Pull request by avakarev**

### CODE REVIEW

**Comment by ndreynolds** [comment]
:thumbsup: LGTM

**Comment by dziemba** [comment]
... but why?

**Comment by avakarev** [comment]
> ... but why?
Because in fact bin/reset_dummy is a regular routine task (ran automatically by CI on each test run). And that's why I think it should be a rake task instead.

**Comment by dziemba** [comment]
So now you have a rake tasks that loads the whole rails env, just to execute two shell commands. How is this better?

**Comment by avakarev** [comment]
It's better because now everyone can find it in the expected place, so it follows the rails conventions for such kind of code/tasks. And because it's run by the robot mainly, I don't care will it take 500ms or 800ms for her to complete the routine. This is my humble opinion.

File: lib/tasks/dummy_reset.rake [hunk 1]
```diff
@@ -0,0 +1,7 @@
+namespace :dummy do
```
Comment (ndreynolds) [inline lib/tasks/dummy_reset.rake]: I guess the `app` namespace is already referencing the dummy app (e.g., `app:db:migrate`).  Maybe we just call it `app:reset` instead.

File: lib/tasks/dummy_reset.rake [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+  desc 'Reset the dummy\'s database, install and apply all migrations.'
+  task reset: :environment do
```
Comment (ndreynolds) [inline lib/tasks/dummy_reset.rake]: Don't think we need the environment dependency here (since it just calls out to the shell).

**Comment by ndreynolds** [comment]
I can see the argument that it belongs in a rake task somewhere.  Can we just remove the unnecessary environment loading and merge this?

**Comment by dziemba** [comment]
Okay fine, go ahead.

**Comment by avakarev** [comment]
Merged with squash, closing.

**Pull request closed without merging**